docker/pom.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-     Copyright 2019 Basis Technology Corp.
+     Copyright 2020 Basis Technology Corp.
```

Comment:
Should copyright be a range of years, like 2019-2020
Ditto with other copyright changes in this PR.

Reply:
Our current checkstyle regex doesn't like ranges.